### PR TITLE
👷⭕ Add CI build system to preview the changes

### PR DESCRIPTION
**Pull request by dpshelio**
:crossed_fingers: it should work as it does with hpc-shell

### CODE REVIEW

**Comment by dpshelio** [comment]
All right, this is finally working! @tkoskela please, review and merge if you are happy. (I've also have included the fix for Travis

**Comment by tkoskela** [comment]
The `home` page does not seem to exist, it returns a blank page with `Artifact not found`

Also I don't know how to review this, you have to request me first? Oh, ok, I can request myself :)

**Comment by tkoskela** [comment]
Otherwise it looks good! Not that I understand a whole lot of what is happening. CircleCI is just checking that every command returns 0? Are some of them just there for debugging?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            cat _extras/snippets_library/UCL_sge/_config_options.yml >> _config.yml
+
+      - run:
+          name: Jekyll build
+          command: bundle exec jekyll build -d html 2> std.err
+
+      - run:
+          name: Check Jekyll build
+          command: |
+            cat std.err
```
Comment (tkoskela) [inline .circleci/config.yml:56]: Can this ever fail or is it just there for debugging?

**Review by tkoskela — APPROVED** [review]

**Pull request merged**